### PR TITLE
Feat/enable collector

### PR DESCRIPTION
**Pull request by jneo8**
depends on #18 
- enable collector on exporter

### CODE REVIEW

File: src/config.py [hunk 1]
```diff
@@ -1,25 +1,57 @@
...
-    "storcli-deb",
-    "perccli-deb",
-    "sas2ircu-bin",
-    "sas3ircu-bin",
-]
-VENDOR_TOOLS = [
-    "ssacli",
-] + TPR_VENDOR_TOOLS
+class SystemVendor(str, Enum):
+    """Different hardward system vendor."""
```
Comment (sudeephb) [inline src/config.py:15]:
```suggestion
    """Different hardware system vendor."""
```

File: src/config.py [hunk 2]
```diff
@@ -1,25 +1,57 @@
...
+
+
+TPR_RESOURCES: t.Dict[HWTool, str] = {
+    HWTool.STORCLI: "storcli-deb",
+    HWTool.PERCCLI: "perccli-deb",
+    HWTool.SAS2IRCU: "sas2ircu-bin",
+    HWTool.SAS3IRCU: "sas3ircu-bin",
+}
+
+EXPORTER_COLLECTOR_MAPPING = {
```
Comment (chanchiwai-ray) [inline src/config.py:46]: I would suggest that rename the collectors to 

```
            "collector.hpe_ssa",
            "collector.ipmi_dcmi",
            "collector.ipmi_sel",
            "collector.ipmi_sensor",
            "collector.lsi_sas_2",
            "collector.lsi_sas_3",
            "collector.mega_raid",
            "collector.poweredge_raid",
```

**Review by chanchiwai-ray — CHANGES_REQUESTED** [review]

File: src/config.py (hunk 2 above)
Comment (jneo8) [reply src/config.py:46]: Is that the required settings we want?

Comment (chanchiwai-ray) [reply src/config.py:46]: Yes

**Review by Pjack — APPROVED** [review]

File: src/config.py (hunk 2 above)
Comment (jneo8) [reply src/config.py:46]: Updated

File: src/config.py (hunk 1 above)
Comment (jneo8) [reply src/config.py]: Updated

**Review by sudeephb — APPROVED** [review]

**Review by chanchiwai-ray — APPROVED** [review]

**Pull request merged**